### PR TITLE
TW-220: Improve for load image first time and keep image when scrolling

### PR DESCRIPTION
**Pull request by nqhhdev**
- Issue: #220 

- Resolved:

https://github.com/linagora/twake-on-matrix/assets/99852347/da687d51-55f7-4dc8-8cba-f96db11f4f1e

### CODE REVIEW

File: lib/config/themes.dart [hunk 1]
```diff
@@ -41,8 +41,11 @@ abstract class FluffyThemes {
   );
 
   static const Duration animationDuration = Duration(milliseconds: 250);
+  static const Duration animationDurationMedium = Duration(milliseconds: 500);
   static const Curve animationCurve = Curves.easeInOut;
 
+  static const String placeholderKey = 'placeholder';
```
Comment (hoangdat) [inline lib/config/themes.dart:47]: move it to MXC file

File: lib/config/themes.dart [hunk 2]
```diff
@@ -41,8 +41,11 @@ abstract class FluffyThemes {
   );
 
   static const Duration animationDuration = Duration(milliseconds: 250);
+  static const Duration animationDurationMedium = Duration(milliseconds: 500);
```
Comment (hoangdat) [inline lib/config/themes.dart:44]: move ti to MXC class. It is duration for MXC, not for all the thing

File: lib/widgets/mxc_image.dart [hunk 3]
```diff
@@ -238,4 +236,8 @@ class _MxcImageState extends State<MxcImage> {
               return placeholder(context);
             }));
   }
+
+  @override
+  // TODO: implement wantKeepAlive
+  bool get wantKeepAlive => true;
```
Comment (hoangdat) [inline lib/widgets/mxc_image.dart:242]: please write a ADR

File: lib/widgets/mxc_image.dart [hunk 4]
```diff
@@ -191,22 +191,19 @@ class _MxcImageState extends State<MxcImage> {
   Widget placeholder(BuildContext context) =>
       widget.placeholder?.call(context) ??
       const Center(
+        key: Key(placeholderKey),
```
Comment (sherlockvn) [inline lib/widgets/mxc_image.dart:194]: I don't understand why you use key here, because key is the same with all image is `placeholder`

**Review by sherlockvn — APPROVED** [review]

**Review by hoangdat — APPROVED** [review]

**Pull request merged**